java/README.md

```diff
@@ -14,13 +14,12 @@ This directory contains the java worker, with the following components.
 
 ```sh
 # build native components
-sh ../build.sh -l java "$@"
+sh ../build.sh -l java
 
 # build java worker
 mvn clean install -Dmaven.test.skip
 
 # test
-export RAY_CONFIG=ray.config.ini
```

Comment:
This should not be removed as people writing stand-alone java apps should have this setup before running.